cli/build.gradle.kts

```diff
@@ -19,6 +19,7 @@
  * License-Filename: LICENSE
  */
 
+val antennaVersion: String by project
```

Comment:
Commit message nit: Please make the `Resolves #3024.` line go between body and footer.